pkg/kubelet/metrics/metrics.go

```diff
@@ -277,6 +279,18 @@ var (
 		[]string{"resource_name"},
 	)
 
+	// PodResourcesEndpointRequestsTotalCount is a Counter that tracks the cumulative number of requests to the PodResource endpoints.
+	// Broken down by server API version.
+	PodResourcesEndpointRequestsTotalCount = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           PodResourcesEndpointRequestsTotalKey,
+			Help:           "Cumulative number of requests to the PodResource endpoint, Broken down by server api version.",
+			StabilityLevel: metrics.STABLE,
```

Comment:
We can't add stable metrics until the metrics stability framework is GA.  Please make this alpha.

Reply:
Addressed!